static/app/components/demo/demoHeader.tsx

```diff
@@ -130,8 +125,6 @@ export default function DemoHeader() {
             organization: null,
           });
         }}
-        target="_blank"
-        rel="noreferrer noopener"
```

Comment:
This button didn't have a `href` so it was already just a regular button with an onclick handler.